onnx/shape_inference.py

```diff
@@ -24,7 +24,7 @@
 bug in shape inference), and the result is unspecified.
 
 Arguments:
-    input (Union[ModelProto, Text, bytes], Text, bool) -> ModelProto
+    input (Union[ModelProto, Text, bytes], Text, bool, bool) -> ModelProto
```

Comment:
what is the Text for?

Reply:
just a nit: correct the comment here that there should be 2 bool (check_type and strict_mode).

Reply:
Oh. I should also remove Text here. Thanks for catching this!